theano/sandbox/rng_mrg.py

```diff
@@ -330,6 +330,7 @@ def R_op(self, inputs, eval_points):
 
 class mrg_uniform(mrg_uniform_base):
     # CPU VERSION
+    _f16_ok = True
```

Comment:
I'm not actually sure it is correct for the case where the perform should accept float16, but the c_code is explicitly disabled for float16.